geode-core/src/main/java/org/apache/geode/admin/internal/ConfigurationParameterImpl.java

```diff
@@ -254,21 +254,9 @@ protected void fireConfigurationParameterValueChanged(ConfigurationParameter par
    */
   protected void setInternalState(String description, Object value, Class type,
       boolean userModifiable) {
-    if (description == null || description.length() == 0) {
```

Comment:
Why can these things now be null?

Reply:
It was impossible to serialize and deserialize a default instance of ConfigurationParameterJMXImpl with this check in place.  We could move these checks to other code but didn't think it was worthwhile for the long-deprecated Admin API.

Reply:
Ok, looks good.